test/image/mocks/mapbox_layers.json

```diff
@@ -539,23 +539,13 @@
           "below": "water",
           "color": "#ece2f0",
           "opacity": 0.8
-        },
-        {
-          "sourcetype": "vector",
-          "source": "mapbox://mapbox.mapbox-terrain-v2",
-          "sourcelayer": "contour",
-          "type": "line",
-          "color": "red",
-          "line": {
-            "width": 2
-          }
         }
       ]
     },
 
     "mapbox2": {
       "domain": {"x": [0.52, 1], "y": [0, 1]},
-      "style": "mapbox://styles/mapbox/light-v10",
+      "style": "light",
```

Comment:
Is it possible to keep this link in our tests? 
Something similar to what you tested [here](https://github.com/plotly/plotly.js/pull/7015#issuecomment-2153992246)? 
It could also be any free style or an asset just for our testing.

Reply:
Sure, the "light" currently refers to the "https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json" , so I inserted that the URL instead.